odc/stats/plugins/lc_level34.py

```diff
@@ -51,9 +49,6 @@ def __init__(
         self.watper_threshold = (
             watper_threshold if watper_threshold is not None else [1, 4, 7, 10]
         )
-        self.water_seasonality_threshold = (
-            water_seasonality_threshold if water_seasonality_threshold else 3
-        )
```

Comment:
the threshold is required, which is 0.25

Reply:
As discussed, this threshold is applied at A1 and the band containing the data is used here directly now i.e. water_season